WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingLaunchStoreView.swift

```diff
@@ -73,13 +80,63 @@ struct StoreOnboardingLaunchStoreView: View {
             }
         }
         .navigationTitle(Localization.title)
+        .task {
+            await viewModel.checkEligibilityToPublishStore()
+        }
+    }
+
+    var freeTrialBanner: some View {
+        HStack(alignment: .top, spacing: Layout.FreeTrialBanner.horizontalSpacing) {
+            Image(uiImage: .infoOutlineImage)
+                .resizable()
+                .renderingMode(.template)
+                .foregroundColor(Color(.withColorStudio(.green, shade: .shade80)))
```

Comment:
Super nit: this color looks really dark in dark mode - should we just use the text color like the banner on the dashboard screen for simplicity and consistency?

Reply:
Done in 52ecd0eea446ac872931e24784ef1988fad12443